packages/grid/x-data-grid-pro/src/hooks/features/columnResize/useGridColumnResize.tsx

```diff
@@ -245,7 +259,7 @@ export const useGridColumnResize = (
       const doc = ownerDocument(apiRef.current.rootElementRef!.current);
       doc.body.style.cursor = 'col-resize';
 
-      separatorSide.current = getSeparatorSide(event.currentTarget);
+      separatorSide.current = getSeparatorSide(event.currentTarget, theme.direction);
```

Comment:
Now makes sense to rename this ref to something like `expansionDirection`, since it doesn't represent the side of the separator anymore.

Reply:
Hmm, good point. Since it's not always an expansion, maybe `resizeDirection`?

Reply:
`resizeDirection` is good.